WordPress/Classes/ViewRelated/Me/MeViewController.m

```diff
@@ -0,0 +1,270 @@
+// Me contents:
+//
+// + (No Title)
+// | Account Settings
+// | Help & Support
+//
+// + (No Title)
+// | Log out
+
+#import "MeViewController.h"
+#import "SettingsViewController.h"
+#import "SupportViewController.h"
+#import "UIImageView+AFNetworkingExtra.h"
+#import "MeHeaderView.h"
+#import "ContextManager.h"
+#import "AccountService.h"
+#import "WPAccount.h"
+#import "LoginViewController.h"
+#import <WordPress-iOS-Shared/WPTableViewCell.h>
+
+const typedef enum {
+    MeRowAccountSettings = 0,
+    MeRowHelp = 1,
+    MeRowLoginLogout = 0
+} MeRow;
+
+const typedef enum {
+    MeSectionGeneralType = 0,
+    MeSectionWpCom
+} MeSectionContentType;
+
+static NSString *const MVCCellReuseIdentifier = @"MVCCellReuseIdentifier";
+
+static CGFloat const MVCTableViewRowHeight = 50.0;
+
+@interface MeViewController () <UITableViewDataSource, UITableViewDelegate, UIActionSheetDelegate>
+
+@property (nonatomic, strong) UITableView *tableView;
+@property (nonatomic, strong) MeHeaderView *headerView;
+
+@end
+
+@implementation MeViewController
+
+- (void)dealloc
+{
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
+}
+
+- (void)loadView
+{
+    [super loadView];
+
+    self.tableView = [[UITableView alloc] initWithFrame:self.view.bounds style:UITableViewStyleGrouped];
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    self.tableView.rowHeight = MVCTableViewRowHeight;
+    self.tableView.translatesAutoresizingMaskIntoConstraints = NO;
+    [self.tableView registerClass:[WPTableViewCell class] forCellReuseIdentifier:MVCCellReuseIdentifier];
+    [self.view addSubview:self.tableView];
+
+    self.headerView = [[MeHeaderView alloc] initWithFrame:CGRectMake(0.0, 0.0, CGRectGetWidth(self.view.bounds), MeHeaderViewHeight)];
+
+    [self setupAutolayoutConstraints];
+}
+
+- (void)setupAutolayoutConstraints
+{
+    NSMutableDictionary *views = [@{@"tableView": self.tableView} mutableCopy];
+    [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|[tableView]|"
```

Comment:
No need to change this, just making a note. The default is horizontal so including the `H:` isn't necessary.